fastlane/Fastfile

```diff
@@ -580,23 +580,27 @@ platform :ios do
       notify_testers: false
     )
 
-    # Install SentryCLI prior to trying to upload dSYMs
-    sh('curl -sL https://sentry.io/get-cli/ | bash')
-
```

Comment:
The fact that CI passes is proof enough that this is unnecessary now, but for reference, this is unnecessary because our CI agents [are provisioned with it already](https://github.com/Automattic/buildkite-ci/blob/29b21405254993f92783c45875bb475a22afcc8f/src/macos-vms/manifests/xcode-13.json#L5).

Reply:
Is this impacting our confidence in being able to run a release build locally in an emergency, if we needed to?

Reply:
Good question – I'd be fine to do something like an `UI.user_error('Sentry not installed') unless sh('command -v sentry-cli')` in a `before` hook to preserve running locally, but IMHO having a `curl $ANYTHING | bash` is generally a bad idea (though if you `blame` this file, I think I'm the one who put this here... 😅 )

Reply:
I like the idea of erroring out if a pre-required external tool is not available, with a suggestion on how to install it, but without doing it for the user.

If we attempted to install it, we'd take on board the responsibility of keeping the install code working and up to date.

Reply:
Addressed in 19e1106b1a12de17987349c42f92f577a65404cb